confidant/models/blind_credential.py

```diff
@@ -16,6 +16,24 @@
 from confidant.models.connection_cls import DDBConnection
 
 
+class NonNullUnicodeSetAttribute(UnicodeSetAttribute):
```

Comment:
Could we extract this into its own module and have the `models.service` and `models.blind_credential` modules import this? So that we don't copy/paste code.

Reply:
sounds right!